### ISSUE TITLE
Ossec.conf takes precedence over agent.conf

### ISSUE THREAD

**Issue by jmv74211** [wazuh/wazuh#4786]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 3.2.1-rev | Logcollector | Agent | Packages | Windows |
| 3.2.4-3254 | Logcollector | Manager | Packages | Centos 7 |

Hello team,

Working on the response to a community user, I have been doing several tests with Wazuh version `3.2.1`, and I have observed the following behavior.

It seems that the following configuration in the `ossec.conf` file

```
  <localfile>
    <location>Security</location>
    <log_format>eventchannel</log_format>
  </localfile>
```

takes precedence over this `agent.conf` configuration

```
<agent_config os="windows">

  <localfile>
    <location>Security</location>
    <log_format>eventchannel</log_format>
    <query>Event/System[EventID != 463]</query>
  </localfile>

</agent_config>
```

In case of removing the configuration indicated in the `ossec.conf `, then it does load the one from the `agent.conf ` file.

This is an unexpected behavior, since the configuration of the agent.conf should take precedence over that of the `ossec.conf`

For further information, [here](https://groups.google.com/forum/#!topic/wazuh/kKCeuq5t6_0) is the thread of the conversation with the user.

Best regards.

**Comment by antoniomanuelfr** [comment]
This behavior was fixed after the 3.3.0 version of wazuh, the PR that fixed this: https://github.com/wazuh/wazuh/pull/402

**Comment by vikman90** [comment]
Thank you @jmv74211 @antoniomanuelfr. Let me close this issue as it's already fixed.

Cheers!
